src/tests/java/htsjdk/samtools/sra/SRAQueryTest.java

```diff
@@ -45,8 +39,6 @@ public void testUnmappedCounts(String acc, int numberUnalignments) {
 
     @Test(dataProvider = "testReferenceAlignedCounts")
     public void testReferenceAlignedCounts(String acc, String reference, int refernceStart, int numberAlignments) {
```

Comment:
type in `refernceStart`; also, most if not all parameters in the test methods can be final.  Since we are touching so much of the code, would it make sense to add in these finishing touches?